system-requirements.md

```diff
@@ -23,6 +23,7 @@ Telerik UI for Blazor targets `netstandard2.1`. Thus, apps and libraries that re
 * `net5.0`
```

Comment:
This bullet list and the paragraph above it are no longer true for version 5.0. We can start with "The latest version of Telerik UI for Blazor targets net6.0" and optionally, clarify that the older target framework can be used with 4.6.0.